project/themes/default/index.php

```diff
@@ -152,18 +152,15 @@
 
```

Comment:
1. Did you test your code on your local machine ? 
2. Which part  or which feature is your code change for ?

Reply:
1, Yes, all the code are tested on my local machine. Our presentation demo is based on these new codes.
2, I mainly change the index.php file and set several divs invisible so our layout looks clean and good for presentation. None of functional part is touched.